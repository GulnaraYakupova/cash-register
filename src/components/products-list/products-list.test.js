import React from 'react';
import ProductsList from './products-list';
import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

const data = [{
    id: '1',
    name: 'Гречка',
    amount: 2,
    price: 150,
  },
  {
    id: '2',
    name: 'Хлеб',
    amount: 2,
    price: 150,
  },
];

it ('Should render ProductsList component', () => {
  const component = mount(
  <ProductsList 
    data={data}
    onProductsItemClick={()=>{}}
    onSearchPanelType={()=>{}}
   />
  );
  expect(component).toMatchSnapshot();
});