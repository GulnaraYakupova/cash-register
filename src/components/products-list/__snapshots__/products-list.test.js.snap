// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ProductsList component 1`] = `ReactWrapper {}`;
